src/xdocs/config_blocks.xml

```diff
@@ -1045,6 +1045,26 @@ allowedFuture.addCallback(result -> {
 &lt;module name=&quot;RightCurly&quot;/&gt;
         </source>
 
+        <p>Example: </p>
+        <div class="wrapper">
+          <pre>
+public void test() {
+  if (foo) {
+    bar;
```

Comment:
How is bar valid java code? All examples should be valid code as that is what users will understand and connect to.